src/components/CampaignsAdmin/edit/CampaignDetails.tsx

```diff
@@ -109,7 +223,7 @@ export function CampaignDetails() {
     }
     const updateLocationConfigResult = await updateLocationConfig(Number(id), campaign?.locationInfo);
     if (updateLocationConfigResult) {
-      photoStatusMessage('Photo uploaded ');
+      StatusMessage('Photo uploaded ');
```

Comment:
Why is this called `StatusMessage` and not `displayStatusMessage` or similar. From a conventions standpoint: 

1. Capital S in Status indicates that the object is a component or a class, not a method. Reading the line without context will make anyone think you're instantiating a new object. Methods in javascript/typescript are written in [camelCase](https://en.wikipedia.org/wiki/Camel_case)
2. `StatusMessage` reads like the name of a class, not an action you're taking. You need a verb of what you're doing